Add webview final

ChrisChinchilla opened the pull request:
@amuramoto Finally clean I hope :)

amuramoto commented:
Hi @ChrisChinchilla - Your PR has a few errors in it:

- README has a link to #. Should link to the blog post.
- Copyright at top of app.js has wrong reference to quick start tutorial and wrong link.
- You have a .env file but app.js only checks process.env (ln 33-35, 106)
- app.js imports crypto but never uses it. It's also not in package.json
- ln 73 you use function for callback, but everywhere else you use arrow function syntax
- dotenv is not in package.json
- In options.html ln 23 you have a hardcoded app id

ChrisChinchilla commented:
@amuramoto Updates pushed and comments below.

* Article not live, so waiting for link to replace the '#'.
* Updated _app.js_, but again need the link to finalise.
* This example uses the dotenv module, which overrides that. It's mentioned in the tutorial too. I will go over and retrofit all articles to use the same env process later.
* 👍 
* Mostly just me still catching up with ES6, updated.
* Yes it is… 🤔 
* Removed, I was trying to find a way of using dotenv in an HTML file, but gave up on the idea, as webpack for one file seemed overkill.

amuramoto commented:
@ChrisChinchilla Cool, thanks. Please close this PR and resend once the links are ready. We shouldn't merge these examples until they are ready to be published.

amuramoto commented on webview/README.md:
```diff
@@ -0,0 +1,20 @@
+# Webview Tutorial - Finished Project
+
+This is the final code for the [Messenger Platform Webview tutorial](#).
```
add link

ChrisChinchilla replied:
Done @amuramoto
